cmis/models/cmis_binding.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class CmisBinding(models.AbstractModel):
+    _name = 'cmis.binding'
+    _inherit = 'external.binding'
+    _description = 'DMS Binding (Abstract)'
+
+    cmis_backend_id = fields.Many2one(
+        'cmis.backend', 'CMIS Backend', required=True,
+        ondelete='restrict', oldname='backend_id')
```

Comment:
What if you want multiple connector types on the same model? IMO the granularity is better